src/main/java/io/stargate/sgv2/jsonapi/exception/catchable/CheckedApiException.java

```diff
@@ -24,4 +31,27 @@ public CheckedApiException(String message, Throwable cause) {
   public CheckedApiException(Throwable cause) {
     super(cause);
   }
+
+  // Add a place to slightly massage value; can be further improved
```

Comment:
Moved these down from sub-class as they need to be shared b/w ToCQLCodecException and ToJSONCodecException